src/lib/styles/global/grid.scss

```diff
@@ -20,3 +20,39 @@
     box-sizing: border-box;
   }
 }
+
+.content-grid {
+  display: flex;
+  flex-direction: column;
+  gap: var(--row-gap);
+  width: 100%;
+
+  @include media.min-width(large) {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    place-items: stretch;
+    column-gap: var(--column-gap);
+    min-height: auto;
+
+    grid-template-rows: auto;
+    grid-template-areas:
+      "item-a item-a item-a item-a item-a item-a item-b item-b item-b item-b item-b item-b"
+      "item-c item-c item-c item-c item-c item-c item-d item-d item-d item-d item-d item-d";
+  }
+
+  .item-a {
```

Comment:
Just because of the `.content-grid`: `.item-a` -> `.content-a`?

Reply:
Like it too! Done.